### DISCUSSION TITLE
Identifying and creating "current rate" sensors.

### DISCUSSION THREAD

**Discussion by Hexadecimalism** [hultenvp/solis-sensor#345] [Q&A]
![image](https://github.com/hultenvp/solis-sensor/assets/14233254/5d4d7e63-cf95-47cc-8702-018c49bc3783)

So I'm trying to create my power dashboard and I can find all the items in blue (not included in blue is the battery charge level which I can also find).

All that works well enough for the display below

![image](https://github.com/hultenvp/solis-sensor/assets/14233254/a9b79e3e-0566-45ab-b3dd-644d572a3f90)

However, I'm working on a custom dashboard and I cannot seem to find the rates at each the panels are generating, power is being consumed, battery is being charged/discharged and the current load for the house. I don't want to "collect" that data, just display (akin to how the soliscloud website does it)  it even if it will always be a few minutes or seconds out of date. Is there a way that it can be obtained by combining and or converting values in other solis sensors into a virtual sensor?

**Comment by jwindley** [comment]
Hi were you able to figure this out?
